init.sh

```diff
@@ -0,0 +1,10 @@
+if [! -d .oh-my-zsh]
```

Comment:
There are some reasons why this won't work:
* Remember that [ is a program, and now [! will be looked up instead. Here you need a space in order for it to work: [ !
* The same goes for the end, .oh-my-zsh and ] need to be to separate arguments to the [ program
* Think about where this script is invoked and how it will interpret the relative link .oh-my-zsh